builder/osc/common/step_source_omi_info_test.go

```diff
@@ -13,22 +14,23 @@ import (
 )
 
 // Create statebag for running test
-func getState() multistep.StateBag {
+func getState() (multistep.StateBag, error) {
 	state := new(multistep.BasicStateBag)
 	accessConfig := &AccessConfig{}
 	accessConfig.RawRegion = "eu-west-2"
 	var oscConn *osc.APIClient
 	var err error
 	if oscConn, err = accessConfig.NewOSCClient(); err != nil {
-		return nil
+		err := fmt.Errorf("error in creating osc Client")
```

Comment:
err, is already set by `accessConfig.NewOSCClient()`, maybe you shouln't unset it here.
Or as last reuse it:
```
 		err := fmt.Errorf("error in creating osc Client: %s", err.Error() )
```